presto-router/src/main/java/com/facebook/presto/router/spec/RouterSpec.java

```diff
@@ -65,4 +73,16 @@ public SchedulerType getSchedulerType()
     {
         return schedulerType.orElse(RANDOM_CHOICE);
     }
+
+    @JsonProperty
+    public URI getPredictorUri()
```

Comment:
return Optional?

Reply:
Thanks for the suggestion! I updated it.